learn/src/main/java/org/navgurukul/learn/ui/learn/adapter/ExerciseSlugAdapter.kt

```diff
@@ -130,18 +132,26 @@ class ExerciseSlugAdapter(callback: (Exercise.ExerciseSlugDetail) -> Unit) :
     }
 
 
-    private fun initDefaultView(
+    private fun initImageView(
         item: Exercise.ExerciseSlugDetail,
         binding: ItemSlugDetailBinding
     ) {
         binding.youtubeView.visibility = View.GONE
         binding.markDownContent.visibility = View.GONE
+        binding.imageView.visibility = View.VISIBLE
 
-        binding.defaultTextView.visibility = View.VISIBLE
-        binding.defaultTextView.text = item.value.toString()
-
+        val value = item.value
+        val gson = Gson()
+        try {
+            val url = gson.fromJson(gson.toJson(value), Image::class.java).url
```

Comment:
objectification of json can be moved to a small extension on string class, would make the code look more concise and elegant